yaptide/batch/set_simulation_state.py

```diff
@@ -0,0 +1,54 @@
+from enum import Enum
+import argparse
+import json
+import logging
+import signal
+import ssl
+from urllib import request
+
+
+def send_request(simulation_id: int, backend_url: str, simulation_state: str):
```

Comment:
This seems to be similar to `send_task_update.py` from watcher:
https://github.com/yaptide/yaptide/blob/f141aa9f07e43ab3b0f3a82dede496a33bda0964/yaptide/batch/watcher.py#L44

Why keep simulation update as separate script and task update as part of watcher. We should have some consistent design.

Reply:
is it good idea to reuse result_sender.py since we already use it to send a post request in collect_script.sh?